vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,521 @@
+plot(out)
+```
+
+```{r plot Rt ebola, echo=FALSE, fig.align = "center", fig.height = 8, fig.width = 8, out.width = "500px", fig.cap="Estimated value of $R_t$ for the incidence data of ebola for Sierra Leone 2014."}
+plot_R0(out)
+```
+
+## Comparison of different epidemiological parameters
+
+The `epiparameter` database stores information on several delay distributions. Parameters for a single delay distribution can be extracted using `epidist()`, but for comparison of all data on a specific pathogen `pathogen_summary()` can be used.
+
+The incubation period and serial interval can be compared for Monkeypox.
+
+```{r, compare monkeyox}
+epidist(pathogen = "monkeypox", delay_dist = "incubation")
+epidist(pathogen = "monkeypox", delay_dist = "serial_interval")
+```
+
+```{r, compare monkeypox tbl}
+pathogen_summary(pathogen = "monkeypox")
+```
+
+Comparison of incubation period, serial interval and onset to hospital admission for ebola.
+
+```{r, compare ebola}
+pathogen_summary(pathogen = "ebola")
+```
+
+## Plotting
+
+The default plotting range for time since infection is from zero to ten days. This can be altered by specifying the `day_range` argument when plotting an `epidist` object.
+
+```{r plotting range epidist, fig.align = "center", fig.height = 8, fig.width = 8, out.width = "500px"}
+plot(sars_cov_incub, day_range = 1:20)
+```
+
+## Planned development for `epiparameter`
+
+-   Currently there is not a function in `epiparameter` convert between the parameters of the Weibull distribution and the mean and standard deviation. This function will come in a future version of the package.
+
+- There is often uncertainty in the estimates of mean, median and percentiles of a parametric distribution, such as the lognormal (e.g. @lessler_incubation_2009). Currently, `epiparameter` does not handle this uncertainty, but future versions will incorporate this and allow the uncertainty to be propagated to tools that use the delay distributions. For example, `EpiEstim` can also estimate the reproduction number when there is uncertainty in the serial interval (see example below), in a future version of `epiparameter` the uncertainty in delay distributions, including serial interval, will be included in the database and will be available to use in these calculations.
+
+```{r, epiestim uncertain SI}
+data("Flu2009")
+
+Rt <- estimate_R(
+  incid = Flu2009$incidence,
+  method = "uncertain_si",
+  config = make_config(list(
+    mean_si = 2.6, 
+    std_mean_si = 1,
+    min_mean_si = 1, 
+    max_mean_si = 4.2,
+    std_si = 1.5, 
+    std_std_si = 0.5,
+    min_std_si = 0.5, 
+    max_std_si = 2.5
+  ))
+)
+```
+
+## Contributing to library of delay distributions {#contributing}
+
+If you would like to contribute to the different delay distributions stored in the `epiparameter` package, you can access the [google sheet](https://docs.google.com/spreadsheets/d/1VzC8NFHSQYPKsiDnxA7gDsm4cVr79outKQZJP8cC7Ro/edit?usp=sharing) and add your data.
```

Comment:
I can view the sheet but I have to request access if I want to edit it. Is that was you're going after?

Reply:
I've updated the settings on the google sheet to enable anyone to edit. Slightly less safe but I think it will just be a lot easier than people having to request editing access. Thanks for pointing this out.